api/proto/info/info.proto

```diff
@@ -431,6 +431,7 @@ message ZInfoDevice {
   // Is there a local_profile from a local_profile_server?
   // The global_profile from the controller is not echoed in this field.
   string local_profile = 45;
+  repeated MaintenanceModeReason maintenance_mode_reasons = 46;
```

Comment:
Please add comment about // deprecated see xxx for the original one.
Also, add a comment who EVE should fill in the deprecated single one (until the time it will be removed).
I assume one of the reasons should be set in the original one.